ProcessMaker/processmaker
# FOUR-19136 | CSS Updates for Screen Templates Panel

# Issue
Ticket: [FOUR-19136](https://processmaker.atlassian.net/browse/FOUR-19136)

This PR adds updated styling to Screen Template Cards in the Screen Templates Panel. Updates include:

- Adds a b-collapse to ScreenTemplateCards.vue that reveals 'Apply Options' when a card is clicked.
- Adds icons and checkboxes for 'Apply Options' (Css, Fields, Layout).
- Adds 'Apply' and 'Cancel' button to each Screen Template Card.

# How to Test
1. Go to branch `observation/FOUR-19136` in `screen-builder` and `processmaker`.
2. Open a Screen in the Screen Builder.
3. Select 'Templates' from the top menu.
    - Screen Template Cards should appear for the templates in your environment.
4. Select a Screen Template Card.
    - The card should expand and the contents of the card should match the [Figma file](https://www.figma.com/design/Jfl3HAOx28TVHMl7cHJNIP/Fall-2024?node-id=641-68784&node-type=frame&t=c0ioyoz7vMiAIuWy-0).
    - Note: if you select one of the options in the card, the 'Apply' button will become enabled.

ci:next
ci:screen-builder:observation/FOUR-19136
ci:deploy

# Code Review Checklist


[FOUR-19136]: https://processmaker.atlassian.net/browse/FOUR-19136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
